test.js

```diff
@@ -0,0 +1,86 @@
+const ethers = require('ethers');
+const crypto = require('crypto');
+const axios = require('axios');
+
+//var id = crypto.randomBytes(32).toString('hex');
+//var privateKey = "0x"+id;
+//console.log("SAVE BUT DO NOT SHARE THIS:", privateKey);
+
+async function test() {
+	let quote, nonce, message, signature;
+
+	const wallet = new ethers.Wallet(process.env.TEST_PRIVATE_KEY);
+	console.log("Address: " + wallet.address);
+
+	// getQuote
+	await axios.post(`http://localhost:8081/getQuote`, {
+		type: "arweave",
+		userAddress: wallet.address,
+		files: [{length: 1256}, {length: 5969}],
+		payment: {
+			chainId: 80001,
+			tokenAddress: "0x0000000000000000000000000000000000001010",
+		},
+	})
+	.then((response) => {
+		quote = response.data;
+		console.log(`Got quote with id: ${quote.quoteId}`);
+	})
+	.catch((error) => {
+		console.error(error);
+		process.exit(1);
+	});
+
+	// upload
+	nonce = Math.floor(new Date().getTime()) / 1000;
+	message = ethers.utils.sha256(ethers.utils.toUtf8Bytes(quote.quoteId + nonce.toString()));
+	signature = await wallet.signMessage(message);
+
+	await axios.post(`http://localhost:8081/upload`, {
+		quoteId: quote.quoteId,
+		files: ["https://example.com/", "https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png"],
+		nonce: nonce,
+		signature: signature,
+	})
+	.then((response) => {
+		console.log("Upload worked");
```

Comment:
"Upload worked" seems misleading here.  This just means that the upload returned 200, and that the server has initiated the payment/upload process.